File: Tests/ObjectDumper.Tests/Testdata/Person.cs
using System;

namespace ObjectDumping.Tests.Testdata
{
    public class Person
    {
        public Person()
        {
            this.GetOnly = 11;
            this.SetOnly = 99;
            this.ByteArray = new byte[] { 1, 2, 3, 4 };
        }

        public string Name { get; set; }

        public char Char { get; set; }

        public int Age { get; set; }

        public int SetOnly { private get; set; }

        public int GetOnly { get; private set; }

        private int Private { get; set; }

        public bool Bool { get; set; }

        public byte Byte { get; set; }

        public byte[] ByteArray { get; set; }

        public sbyte SByte { get; set; }

        public float Float { get; set; }

        public uint Uint { get; set; }

        public long Long { get; set; }

        public ulong ULong { get; set; }

        public short Short { get; set; }

        public ushort UShort { get; set; }

        public decimal Decimal { get; set; }

        public double Double { get; set; }

        public DateTime DateTime { get; set; }

        public DateTime? NullableDateTime { get; set; }

        public DateTimeKind Enum { get; set; }
    }
}

File: Samples/ObjectDumperSample.Netfx.VB/Person.vb
Namespace ObjectDumperSample.Netfx.VB
    Public Class Person

        Public Sub New()

            Me.GetOnly = 11
            Me.SetOnly = 99
            Me.ByteArray = New Byte[] { 1, 2, 3, 4 }
 End Sub

        Public Name As String

        Public Property [char] As Char

        Public Property Age As Integer

        Public Property SetOnly As Integer

        Public Property GetOnly As Integer

        Private Property [Private] As Integer

        Public Property [Bool] As Boolean

        Public Property [Byte] e As Byte

        Public Property ByteArray As Byte()

        Public Property [SByte] As SByte

        Public Property [Float] As Decimal

        Public Property [Uint] As UInteger

        Public Property [Long] As Long

        Public Property [ULong] As ULong

        Public Property [Short] As Short

        Public Property [UShort] As UShort

        Public Property [Decimal] As Decimal

        Public Property [Double] As Double

        Public Property [DateTime] As DateTime

        Public Property NullableDateTime As DateTime?

        Public Property [Enum] As DateTimeKind
    End Class
End Namespace
